.docker/archlinux-docker/rootfs/arm64v8/etc/pacman.d/mirrorlist

```diff
@@ -0,0 +1,22 @@
+# Server list generated by rankmirrors on 2020-04-17
```

Comment:
maybe we create a new repo for this fork. IM weary of adding it directly in the sdk.

Reply:
because of the mirrorlist?

I could update it.  The reason that I'd like to put it here is that if it is here, we build a complete multiplatform development environmentn in the cosmos sdk each hour and in the future developers (like me) will not have to run around the internet trying to figure out where the db build system image is built (oop, it was tmdb's or wait was it iavl and who is tendermint-dev, anyway?)